lib/booking_bug/api.rb

```diff
@@ -0,0 +1,40 @@
+class BookingBug
+  class UnableToAuthenticate < StandardError; end
+
+  class Api
+    def initialize(company_id: Config.booking_bug.company_id)
```

Comment:
`Config` should be declared somewhere in the `BookingBug` namespace. We're currently coupled to an object defined by the application using this library.